packages/webpack-config-flag-plugin/test/fixtures/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
 		sourceType: 'module',
 	},
 	rules: {
+		'import/no-extraneous-dependencies': 'off',
```

Comment:
Test fixtures for this plugin do a lot of imports of `config` and `calypso/config` (converting between those is the point of this plugin).